src/thriveopedia/pages/ThriveopediaHomePage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Godot;
+
+/// <summary>
+///   Thriveopedia page displaying welcome information and links to websites.
+/// </summary>
+public class ThriveopediaHomePage : ThriveopediaPage
+{
+    public override string PageName => "Home";
```

Comment:
This is like an internal name, right? If so this should be lower camelCase to be consistent with the rest of the game.

Reply:
It's used as part of the name for the scene loaded for each page, so needs to be PascalCase there.

Reply:
I did eventually get that far in the code. Saying that explicitly in the base abstract class xml documentation would be nice.